js/src/redux/providers/balancesActions.js

```diff
@@ -144,34 +144,37 @@ export function updateTokensFilter (_addresses, _tokens, options = {}) {
       promises.push(api.eth.uninstallFilter(tokensFilter.filterToId));
     }
 
-    const promise = Promise.all(promises);
-
-    const topicsFrom = [ TRANSFER_SIGNATURE, addresses, null ];
-    const topicsTo = [ TRANSFER_SIGNATURE, null, addresses ];
-
-    const filterOptions = {
-      fromBlock: 0,
-      toBlock: 'pending',
-      address: tokenAddresses
-    };
-
-    const optionsFrom = {
-      ...filterOptions,
-      topics: topicsFrom
-    };
-
-    const optionsTo = {
-      ...filterOptions,
-      topics: topicsTo
-    };
-
-    const newFilters = Promise.all([
-      api.eth.newFilter(optionsFrom),
-      api.eth.newFilter(optionsTo)
-    ]);
-
-    promise
-      .then(() => newFilters)
+    Promise
+      .all([
+        api.eth.blockNumber()
```

Comment:
So this doesn't get triggered on new block, only when the filters need to be updated, which is when you need to update the tokens value of the displayed addresses, ie. when switching views between your account, your address book, contracts, etc. We don't update the tokens balances for accounts we don't show, except those we own (so we can get a notification)